kola/tests/misc/nvidia.go

```diff
@@ -0,0 +1,52 @@
+package misc
+
+import (
+	"bytes"
+	"fmt"
+	"time"
+
+	"github.com/flatcar/mantle/kola/cluster"
+	"github.com/flatcar/mantle/kola/register"
+	"github.com/flatcar/mantle/util"
+)
+
+const (
+	CmdTimeout = time.Second * 300
+)
+
+func init() {
+	register.Register(&register.Test{
+		Name:        "cl.misc.nvidia",
+		Run:         verifyNvidiaInstallation,
+		ClusterSize: 1,
+		Distros:     []string{"cl"},
+		// This test is to test the NVIDIA installation, limited to AZURE for now
+		Platforms:     []string{"azure"},
+		Architectures: []string{"amd64"},
```

Comment:
What instance type does this need? I think we would have to skip this by default and invoke it similar to the cl.internet tests that run on various instance types.